### ISSUE TITLE
[BUG] Unable to Obsfucate if the script uses the ROBLOX Buffer Library

### ISSUE THREAD

**Issue by LostTray** [prometheus-lua/Prometheus#161]
**Describe the bug**
Unable to Obsfucate if you use the buffer Library

**To Reproduce**
`local ABuffer = buffer.create(1048)` Trying to obsfucate this, fails.

**Comment by Brohammer5** [comment]
Your are using LuaU which is not fully supported however you should use the LuaU option as this might be a fix for your problem.

**Comment by LostTray** [comment]
I already use the LuaU option. Is it planned to update the LuaU option with the Buffer Library? or am I going to need to find another Obsfucator?

**Comment by Brohammer5** [comment]
Not sure what the owner plans to do but it mostly likely should be supported in the near future with LuaU being fully supported.

**Comment by nacho00112** [comment]
I don't think so, the author probably abandoned the project, and probably won't return for a long time if he ever returns.

**Comment by nacho00112** [comment]
Someone might make some PR or create an independent fork of this project.

**Comment by Brohammer5** [comment]
Not exactly true as the author has stated that he is busy and doesn't always have free time to focus on this.

**Comment by Zaenalos** [comment]
That's right.
